src/main/java/org/broadinstitute/hellbender/tools/funcotator/engine/FuncotationEngine.java

```diff
@@ -0,0 +1,108 @@
+package org.broadinstitute.hellbender.tools.funcotator.engine;
```

Comment:
I don't think it's worth introducing a whole new sub-package for this class -- I'd put it in the main `funcotator` package instead.

Reply:
Fixed!